### PR TITLE
Fixed all the 9 redirect errors on build #13

### PR DESCRIPTION
**Pull request by najela**
Fixed all the 9 redirect errors shown at http://jenkins.jquery.com/view/Websites/job/stage.api.qunitjs.com/13/console

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "api.qunitjs.com",
   "title": "QUnit API Docs",
   "description": "API reference documentation for QUnit",
-  "version": "1.17.1",
+  "version": "1.17.2",
```
Comment (arschmitz) [inline package.json]: We do version bumps on the websites through `npm version patch` to also create a matching tag and pushing the tag deploys the website to production. So we don't  bump version numbers in PR's

Comment (najela) [inline package.json]: Thanks for the info @arschmitz I had no idea

File: entries/notStrictEqual.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
 	</desc>
 	<longdesc>
 		<p>The <code>notStrictEqual</code> assertion uses the strict inverted comparison operator (<code>!==</code>) to compare the actual and expected arguments. When they aren't equal, the assertion passes; otherwise, it fails. When it fails, both actual and expected values are displayed in the test result, in addition to a given message.</p>
-		<p><a href="/equal"><code>equal()</code></a> can be used to test equality.</p>
+		<p><a href="/equal/"><code>equal()</code></a> can be used to test equality.</p>
 		<p><a href="/strictEqual"><code>strictEqual()</code></a> can be used to test strict equality.</p>
```
Comment (jzaefferer) [inline entries/notStrictEqual.xml]: Looks like you missed this one. Should do a search for all `<a>` elements inside entries, then checking each that doesn't end the href with a forward slash.

File: package.json (hunk 1 above)
Comment (jzaefferer) [inline package.json]: We can just cherry-pick the one commit, no big deal.

File: entries/propEqual.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@
 	<longdesc>
 		<p>The <code>propEqual()</code> assertion provides strictly (<code>===</code>) comparison of Object properties. Unlike <code>deepEqual()</code>, this assertion can be used to compare two objects made with different constructors and prototype.</p>
 		<p><a href="/strictEqual"><code>strictEqual()</code></a> can be used to test strict equality.</p>
```
Comment (arthurvr) [inline entries/propEqual.xml]: Do this one as well?

Comment (najela) [inline entries/propEqual.xml]: @arthurvr  @arschmitz  The commit 4ccbf0e1dc solves all the remaining broke links together with the issues with versioning

**Comment by najela** [comment]
@arthurvr @arschmitz The commit 4ccbf0e solves all the remaining broke links together with the issues with versioning

**Comment by leobalter** [comment]
LGTM.

++ to the grep command.

**Comment by arthurvr** [comment]
Thanks. :+1:

**Pull request closed without merging**